test/integration/price-import.spec.coffee

```diff
@@ -87,16 +87,24 @@ describe 'Price Importer integration tests', ->
     prices = [
       {
         sku: 'sku1'
-        value:
-          centAmount: 9999
-          currencyCode: 'EUR'
+        prices: [
+          {
+            value:
+              centAmount: 9999
+              currencyCode: 'EUR'
+          }
+        ]
```

Comment:
I think it may be more flexible to have prices per variant then per product. Not many external systems can bundle variants under common product object.